api/src/controllers/profile.ts

```diff
@@ -105,16 +107,22 @@ export const updateProfileContacts = async (
 
     const provisionerRelatedChanges = editCompares.some(editCompare => editCompare);
     if (provisionerRelatedChanges) {
-      await requestProfileContactsEdit(Number(profileId), contacts);
+      const request = await requestProfileContactsEdit(Number(profileId), contacts, user, false);
```

Comment:
Is there a way we could change the last parameter so we don't directly pass boolean?